content/synthetics/browser_check.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Browser Check
+kind: documentation
+beta: true
+description: Simulate and monitor user journeys from specific locations.
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/uptime_check"
+  tag: "Documentation"
+  text: "Configure an Uptime Check"
+---
+
+<div class="alert alert-warning">Synthetics is in beta for the Datadog US Site. Request access via the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics request form</a>.</div>
```

Comment:
Should we also add a link to the form for EU based accounts (https://app.datadoghq.eu/synthetics/beta) or is it useless?

Reply:
Useless: we don't support EU orgs for now.